src/pages/partners/components/SponsorDivider.js

```diff
@@ -0,0 +1,8 @@
+import { Divider } from "@mui/material";
+import React from "react";
+
+const SponsorDivider = () => {
```

Comment:
Why do we need this? Why don't we just put the Divider straight in the Sponsor?

Reply:
I'm not sure myself. Gotta ask myself from 6 months ago.

Reply:
Resolved